### PR TITLE
style: update staking page

### PR DESCRIPTION
**Pull request by jincan39**
## Description

closes: #645

---

Before we can merge this PR, please make sure that all the following items have been checked off. If any of the checklist items are not applicable, please leave them but write a little note why.

- [x] Linked to Github issue with discussion and accepted design OR have an explanation in the PR that describes this work
- [x] Re-reviewed files changed in the Github PR explorer

## If PR to `staging`
- [x] Updated relevant documentation in the code
- [x] Test code changes manually, and describe your procedure in a comment on this PR
- [x] Mark this PR with the correct milestone

### CODE REVIEW

**Comment by Kevingislason** [comment]
It seems the components at the top of the page ("Total balance" / "Resources")  are now a bit more narrow than the tables farther down the page. I don't see this on the current staging branch. Do you see this also? Not immediately sure what would change this from looking at your code changes.


Screen Shot 2023-03-01 at 11 06 04 PM

**Comment by jincan39** [comment]
> It seems the components at the top of the page ("Total balance" / "Resources") are now a bit more narrow than the tables farther down the page. I don't see this on the current staging branch. Do you see this also? Not immediately sure what would change this from looking at your code changes.
> 
> Screen Shot 2023-03-01 at 11 06 04 PM

yes good notice the whole block's width changed after replaced with smaller button & font size. confirming with yeye.

File: src/components/Footer/index.tsx [hunk 1]
```diff
@@ -6,25 +6,24 @@ import {
...
           Calamari Network is a private layer built for the entire Kusama
           ecosystem. Built on the substrate framework, Calamari Network is
           natively compatible with other projects and parachain assets including
           wrapped major cryptoassets.
         </div>
-        <div className="flex items-center text-xl text-secondary">
+        <div className="flex items-center text-white-80%">
           <span>Made with</span>
-          <Icon className="mx-3 inline w-5 align-middle" name="heart" />
+          &nbsp;❤️&nbsp;
```
[the remark this answers is not in the document]
Comment (jasonintju) [reply src/components/Footer/index.tsx:23]: never mind, just do as designer told u

**Review by jasonintju — APPROVED** [review]

**Comment by jincan39** [comment]
Finished yeye's checks.

**Review by Kevingislason — APPROVED** [review]

**Pull request merged**